src/components/Tooltip/story.tsx

```diff
@@ -62,74 +54,62 @@ const Element: React.FC<React.HTMLAttributes<HTMLDivElement>> = (props) => (
   <div className="mr-5" {...props} />
 );
 
-const Overview = () => {
```

Comment:
When this component was defined outside the story's render function, knobs didn't work.